src/ui/Pricing/SubscriptionSettings/index.svelte

```diff
@@ -86,10 +86,10 @@
   </Setting>
 
   {#if subscription && !isCanceled && !isFree}
-    <Setting>
+    <Setting class="$style.cancel body-2 justify">
```

Comment:
`body-2` class breaks existing desktop UI. Please, make sure it's consistent with Figma design